tripy/tripy/frontend/trace/ops/gather.py

```diff
@@ -110,10 +111,10 @@ def gather(input: "tripy.Tensor", dim: int, index: "tripy.Tensor") -> "tripy.Ten
         :linenos:
         :caption: Example
 
-        data = tp.iota((3, 2, 2))
+        data = tp.iota((3, 3, 2))
         indices = tp.Tensor([0, 2], dtype=tp.int32)
-        output = tp.gather(data, 0, indices)
+        output = tp.gather(data, 1, indices)
 
-        assert np.array_equal(cp.from_dlpack(output).get(), np.take(cp.from_dlpack(data).get(), cp.from_dlpack(indices).get(), axis=0))
+        assert np.array_equal(cp.from_dlpack(output).get(), np.take(cp.from_dlpack(data).get(), cp.from_dlpack(indices).get(), axis=1))
```

Comment:
We should write a directed test instead of changing the example IMO

Reply:
I filed https://github.com/NVIDIA/TensorRT-Incubator/issues/52 to add a lot more exhaustive tests.